itests/dummydeal_test.go

```diff
@@ -105,29 +107,61 @@ func TestDummydeal(t *testing.T) {
 	f.start()
 
 	// Create a CAR file
-	randomFilepath, err := testutil.CreateRandomFile(5, 2000000)
+	tempdir := os.TempDir()
```

Comment:
If you use `t.TempDir()` it will automatically get cleaned up:
```
// TempDir returns a temporary directory for the test to use.
// The directory is automatically removed by Cleanup when the test and
// all its subtests complete.
// Each subsequent call to t.TempDir returns a unique directory;
// if the directory creation fails, TempDir terminates the test by calling Fatal.
```

Reply:
OK, removing the `t.Cleanup` function then!